packages/manager/apps/pci-kubernetes/src/components/create/BillingStep.component.tsx

```diff
@@ -66,173 +57,128 @@ export default function BillingStep(props: TBillingStepProps): ReactElement {
   const savingsPlanUrl = `${projectURL}/savings-plan`;
   const showSavingPlan = useSavingsPlanAvailable();
 
-  const calculatePrice = (basePrice: number | null, zonesNumber?: number | null): number => {
+  const calculatePrice = (basePrice: number | null, zonesNumber = 0, optionalPrice = 0): number => {
     const price = Number(basePrice);
-    return zonesNumber ? zonesNumber * price : price;
+    const base = zonesNumber ? zonesNumber * price : price;
+    return base + (zonesNumber ? zonesNumber * optionalPrice : optionalPrice);
   };
-  const computedPrice = calculatePrice(props.price, props.selectedAvailabilityZonesNumber);
+  const computedPrice = calculatePrice(
```

Comment:
Suggestions:
1. this can be place outside the component
2. I think this is the same but simpler :
```suggestion
  const calculatePrice = (basePrice: number | null, zonesNumber = 1, optionalPrice = 0): number => {
    const price = Number(basePrice);
    return zonesNumber * (price + optionalPrice);
  };
  
  const computedPrice = calculatePrice(
```

Reply:
I don't know why the suggestion is weird like that